app/models/miq_worker/container_common.rb

```diff
@@ -19,6 +23,10 @@ def scale_deployment
       delete_container_objects if self.class.workers.zero?
     end
 
+    def zone_selector
+      {"#{Vmdb::Appliance.PRODUCT_NAME.downcase}/zone" => MiqServer.my_zone}
```

Comment:
@Fryguy what exactly do we want to do with the nodes? Should they only be a part of a single zone? That is how this implementation would work.

I think I could make this selector more flexible, but do we think we have the use case of more than one node in a zone?

Reply:
Don't use key: value notation for this. Something like `manageiq/zone/<name>=true`. This will allow a node to be in multiple zones.